spec/Body/Body.vspec

```diff
@@ -242,7 +242,7 @@ Lights.DirectionIndicator:
 ##
 Mirrors:
   type: branch
-  instances: ["Left", "Right"]
+  instances: ["Driver", "Passenger"]
```

Comment:
If we go for this we could possibly consider redefining it for motorbikes as left/right in [the motorbike profile](https://github.com/COVESA/vehicle_signal_specification/blob/master/overlays/profiles/motorbike.vspec)